### PR TITLE
New site sidebar

### PR DESCRIPTION
**Pull request by dayhaysoos**
This PR will be adding theming properly to the docusaurus site as well as the new side bar implementation

### CODE REVIEW

**Comment by angiejones** [comment]
are we able to add:

- the texture to the background
- `Docs` label
- the small titles above sections (`SDKs`, `Resources`)?
- Resources section
- language selector
- icons before each label

image

**Comment by dayhaysoos** [comment]
> are we able to add:
> 
> * the texture to the background
> * `Docs` label
> * the small titles above sections (`SDKs`, `Resources`)?
> * Resources section
> * language selector
> * icons before each label
> 
> image

Hey Angie, thank you for the feedback but none of this is ready to review at all. I'm still trying to figure out how to work the side bars to reflect the designs 😅

I should be getting to all those things soon though!

File: site-new/static/img/logo.svg
Comment (angiejones) [inline site-new/static/img/logo.svg:1]: we want to use the same logo that's on the current site, not the tbd{dev} logo

File: site-new/sidebars.ts [hunk 1]
```diff
@@ -12,8 +15,67 @@ import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
...
+      items: [
+        {
+          type: "link",
+          label: "Web5 SDK",
+          href: "web5-sdk",
+          className: "web5-icon",
+        },
+        {
+          type: "link",
+          label: "tbDex SDK",
```
Comment (angiejones) [inline site-new/sidebars.ts:38]:
```suggestion
          label: "tbDEX SDK",
```

The correct capitalization is lowercase `tb` and uppercase `DEX`: `tbDEX`

File: site-new/sidebars.ts [hunk 2]
```diff
@@ -12,8 +15,67 @@ import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
...
+      type: "autogenerated",
+      dirName: "web5-sdk/dids",
+      customProps: { sidebarHeader: "test" },
+    },
+  ],
+  tbdexSidebar: [
+    {
+      type: "doc",
+      id: "tbdex-sdk/index",
+      customProps: { sidebarHeader: "tbDex SDK" },
```
Comment (angiejones) [inline site-new/sidebars.ts:66]:
```suggestion
      customProps: { sidebarHeader: "tbDEX SDK" },
```

**Pull request merged**